config.json

```diff
@@ -41,6 +41,17 @@
         "strings"
       ]
     },
+    {
+      "slug": "darts",
+      "uuid": "4d400a44-b190-4a0c-affb-99fad8ea18da",
+      "core": false,
+      "unlocked_by": "two-fer",
+      "difficulty": 1,
```

Comment:
I think this involves a bit more logic than the other difficulty 1 exercises so maybe it should have difficulty 2? 🙂 

And since it has difficulty 4 or lower it should have a starter implementation (see the [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations)) 🙂

Reply:
Thank you, I will add the starter implementation and change the difficulty level.